Framework/API/src/FrameworkManager.cpp

```diff
@@ -22,10 +22,22 @@
 #include <nexus/NeXusFile.hpp>
 
 #include <Poco/ActiveResult.h>
-#include <tbb/task_scheduler_init.h>
 
 #include <clocale>
 #include <cstdarg>
+#include <memory>
+
+// The below can be replaced with global_control when all platforms
+// have 2019U4
+#include "tbb/tbb_stddef.h"
+
+#if TBB_INTERFACE_VERSION_MAJOR < 11
```

Comment:
Can we use `__has_include(<tbb/global_control.h>)` now?

Reply:
I had a look at this, however Ubuntu has `tbb/global_control.h`but also requires us to do something like `#define PREVIEW_GLOBAL_CONTROL`

By marking it as TBB version 11+ we avoid having to mess with pre-processor defines too

Reply:
Sounds fair. Thanks.